internal/services/oracle/oracle_exadata_infrastructure_data_source.go

```diff
@@ -443,6 +459,7 @@ func (d ExadataInfraDataSource) Read() sdk.ResourceFunc {
 					state.DbServerVersion = pointer.From(props.DbServerVersion)
 					state.DisplayName = props.DisplayName
 					state.EstimatedPatchingTime = FlattenEstimatedPatchingTimes(props.EstimatedPatchingTime)
+					state.ExascaleConfig = FlattenExascaleConfig(props.ExascaleConfig)
```

Comment:
will these properties be used as a data source? e.g.: be referenced by other resources